### PR TITLE
Usage of account.getMedia()

### LINKED ISSUE #693: Use Account.getMedia(size) everywhere
It's new and should be used, instead of the long `ng-src`s.

It automatically returns default image if the image doesn't exist.

Check #659 for usage.

# Account

## Sizes

```js
    /**
     * @type {object}
     */
    this.media = {
      full: self.get.media.picture,
      medium: self.get.media.thumbnail_256x256,
      small: self.get.media.thumbnail_128x128,
      tiny: self.get.media.thumbnail_48x48
    };
```

## getMedia 

```js
    /**
     * @desc Get media or default picture
     * @type {function}
     *
     * @param {string} size
     *
     * @returns {string} URL of the image
     */
    this.getMedia = function (size) {
      if (self.media[size]) {
        return self.media[size];
      }
      return "assets/img/avatar.png";
    };
```

### PR DESCRIPTION
**Pull request by ArsalanSavand**
# Bug fixes
Fixes #693

### CODE REVIEW

File: app/account/user/user_controller.js [hunk 1]
```diff
@@ -130,5 +131,6 @@ UserController.$inject = [
   "DolphinService",
   "Upload",
   "ENV",
-  "toaster"
+  "toaster",
+  "Account"
```
Comment (AmirSavand) [inline app/account/user/user_controller.js]: Before lowercase deps.

Comment (AmirSavand) [inline app/account/user/user_controller.js]: Classes should be injected before other deps, so move it to Upload

File: app/account/user/user_view.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
           <div class="col-sm-4">
             <div class="avatar avatar-outside">
               <span ng-if="!user.get.media.picture">{{ user.getDisplayName()[0] }}</span>
-              <img ng-if="user.get.media.picture" class="img-circle" ng-src="{{ user.get.media.thumbnail_128x128 }}" />
+              <img ng-if="user.get.media.picture" class="img-circle" ng-src="{{ user.getMedia('small') }}" />
```
Comment (AmirSavand) [inline app/account/user/user_view.html]: Use `user.hasMedia` instead of `user.get.media.picture`.

File: app/account/user/user_view.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
           <div class="col-sm-4">
             <div class="avatar avatar-outside">
               <span ng-if="!user.get.media.picture">{{ user.getDisplayName()[0] }}</span>
```
Comment (AmirSavand) [inline app/account/user/user_view.html]: Use `user.hasMedia` instead of `user.get.media.picture`.

File: app/team/team_modal/team_modal_view.html [hunk 4]
```diff
@@ -7,7 +7,10 @@
       <div class="modal-body">
         <!-- Cover -->
         <div class="cover" ng-class="{open: open}" ng-click="open = !open" 
-          ng-init="open = false" style="background-image: url({{ team.user.getMedia('medium') }})">
+          ng-init="open = false"
+          style="background-image: url({{
+              !team.isPending ? team.user.getMedia('medium') : 'assets/img/avatar.png' }}
+            )">
```
Comment (AmirSavand) [inline app/team/team_modal/team_modal_view.html]: Fix indention.

File: app/account/user/user_view.html (hunk 3 above)
Comment (ArsalanSavand) [inline app/account/user/user_view.html]: Done

File: app/account/user/user_view.html (hunk 2 above)
Comment (ArsalanSavand) [inline app/account/user/user_view.html]: Done

File: app/team/team_modal/team_modal_view.html (hunk 4 above)
Comment (ArsalanSavand) [inline app/team/team_modal/team_modal_view.html]: Done

**Pull request merged**